### ISSUE TITLE
NTR: Bone marrow: cystine crystals (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9136]
New term request Bone marrow: cystine crystals (elthat_100520035224):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6001033
- label: Bone marrow cystine accumulation
- parents: [Abnormal bone marrow cell morphology (HP:0005561)](https://hpo.jax.org/app/browse/term/HP:0005561)
- pmid: PMID:28871612
- def: The presence of cystine crystals in bone marrow cells as demonstrated in a fresh bone marrow aspiration sample.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9136
- synonyms:
